From 3da232f71ceaf7e7f5ffe2876320d483470d36c3 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:53 -0400
Subject: [redhat] kabi: Adding symbol clear_user (arch/x86/lib/usercopy_64.c)

Message-id: <3128d45b70370048b4794c73d3b24af5dfd14355.1410510773.git.poros@redhat.com>
Patchwork-id: 96015
O-Subject: [PATCH RHEL7.1 053/233] kabi: Adding symbol clear_user (arch/x86/lib/usercopy_64.c)
Bugzilla: 1071097
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1071097
    https://bugzilla.redhat.com/show_bug.cgi?id=1071097

    Description:
    ============
    Zero a block of memory in user space.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/clear_user

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_x86_64/clear_user b/redhat/kabi/kabi-module/kabi_x86_64/clear_user
new file mode 100644
index 0000000..1e16641
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/clear_user
@@ -0,0 +1,2 @@
+#1-
+0x7da99cdd clear_user vmlinux EXPORT_SYMBOL
-- 
1.7.1